eclipse/che-plugin-registry
# EditorConfig plugin v0.14.4

Signed-off-by: Vitaliy Gulyy <vgulyy@redhat.com>



![](https://open-vsx.org/api/EditorConfig/EditorConfig/0.14.4/file/EditorConfig_icon.png)

### What does this PR do?
Introduces a plugin with EditorConfig VS Code extension v0.14.4

As the EditorConfig extension [was removed](https://github.com/eclipse/che-theia/pull/912) from the default extensions, it was decided to [add it as a separate plugin](https://github.com/eclipse/che/issues/18297).

Additional info:
- https://editorconfig.org/
- https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig
- https://open-vsx.org/extension/EditorConfig/EditorConfig

### Screenshot/screencast of this PR


### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18297

### How to test this PR?

Devfile to test
```
---
    apiVersion: 1.0.0
    metadata:
      generateName: java-web-spring-
    projects:
      -
        name: java-web-spring
        source:
          type: git
          location: "https://github.com/spring-projects/spring-petclinic.git"
          commitId: 4953f87917b449a404a7f1f4e2457836b9eafbbc
    components:
      # Che-Theia without EditorConfig default extension
      # Editor meta.yaml was taken from the PR https://github.com/eclipse/che-theia/pull/912
      -
        type: cheEditor
        alias: che-theia
        reference: >-
            https://raw.githubusercontent.com/chepullreq4/pr-check-files/master/che-theia/pr-912/che_theia_meta.yaml
        memoryLimit: 768Mi
      # EditorConfig Che-Plugin
      -
        type: chePlugin
        reference: >-
            https://raw.githubusercontent.com/eclipse/che-plugin-registry/editorconfig/v3/plugins/EditorConfig/EditorConfig/0.14.4/meta.yaml
      -
        type: chePlugin
        id: redhat/java/latest
        memoryLimit: 1512Mi
        preferences:
          java.server.launchMode: Standard
```

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
